### ISSUE TITLE
scheduler: restarting completed workflows

### ISSUE THREAD

**Issue by oliver-sanders** [cylc/cylc-flow#5474]
* Make it easier to restart a completed workflow.
* Make it more obvious why a completed workflow shuts down.
* Make it easier to extend a completed workflow run (i.e. by triggering extra stuffs).

Related to:

* https://github.com/cylc/cylc-flow/pull/5231
* https://github.com/cylc/cylc-flow/pull/5084
* https://github.com/cylc/cylc-flow/issues/5416

**Comment by hjoliver** [comment]
>   Make it easier to restart a completed workflow.

Is the existing timeout PR not sufficient in your opinion? #5231 

>    Make it more obvious why a completed workflow shuts down.

Agreed, at the least we can log an explanatory message.

 >   Make it easier to extend a completed workflow run (i.e. by triggering extra stuffs).

How is this bullet point different from the first one?

**Comment by oliver-sanders** [comment]
> Is the existing timeout PR not sufficient in your opinion? https://github.com/cylc/cylc-flow/pull/5231

Yes, I've put this up as an umbrella issue to make it easier to track the larger functionality.

> How is this bullet point different from the first one?

Extending a workflow run e.g. by adding a new cycle of tasks is not completely solved by #5231 because it requires the user to know exactly which tasks they need to trigger to kick off the new cycle. The suggestions in #5416 would make this much more user-friendly.

**Comment by oliver-sanders** [comment]
Suggestions made in https://github.com/cylc/cylc-flow/issues/6273

* Increase PT2M restart timer to say PT10M
* Add a new workflow status message for "run to completion"
